### PR TITLE
Gusyatin Kirill - Homework/9

### PR DESCRIPTION
**Pull request by kikill95**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -17,16 +17,33 @@
...
-$(document).on('keyup', '#_search', function(){
+$(document).on('keyup', '#_search', function() {
   $(this.form).trigger('submit.rails');
-})
+});
+
+$(document).on('change', '.quantity input', function () {
+    var $input = $(this);
+    $.ajax({
+        type: 'PUT',
```
Comment (galulex) [inline app/assets/javascripts/application.js]: we do not need to send the ajax call to calculate the sum. just calculate the sum here

Comment (kikill95) [inline app/assets/javascripts/application.js]: In which way should I refactor this?
Should the 'quantity' property be in session's hash?

Comment (galulex) [inline app/assets/javascripts/application.js]: set the `data-attribute` to `quantity` input

`<input type='number' data-price='44.2'>`

on change take the value of each filed and multiply on price and then sum everything

Comment (kikill95) [inline app/assets/javascripts/application.js]: Okay, I will refactor it soon. Thank you for feedback!

**Pull request closed without merging**